parsley/shared/src/main/scala/parsley/internal/deepembedding/frontend/IterativeEmbedding.scala

```diff
@@ -10,15 +10,33 @@ import scala.collection.{mutable, Factory}
 import parsley.internal.deepembedding.ContOps, ContOps.{suspend, ContAdapter}
 import parsley.internal.deepembedding.backend, backend.StrictParsley
 
-private [parsley] final class Many[A, C](init: LazyParsley[mutable.Builder[A, C]], p: =>LazyParsley[A], private [parsley] var debugName: String)
-    extends Binary[mutable.Builder[A, C], A, C](init, p) {
+/**
+  * This trait represents combinators that are iterative, that is, they execute 
+  * parsers multiple times until they cannot match any more
+  * 
+  * @see [[parsley.internal.deepembedding.frontend.debug.TaggedWith `TaggedWith`]] for how this trait is used to bubble up knowledge of
+  * iterative subparsers to the [[parsley.debug.DebugTree `DebugTree`]]
+  * 
+  * @example [[Many]]
```

Comment:
can't do this because `private [parsley]`, the doc engine is upset because there isn't a physically html page that would correspond to `Many`.

Reply:
The examples are in a sense unnecessary, this is a `sealed` trait, they very much all live here

Reply:
oh also the example fields are mainly for code snippets, you probably meant `@see`, but as I said, not really necessary